src/msw/dcmemory.cpp

```diff
@@ -143,9 +143,12 @@ void wxMemoryDCImpl::DoSelect( const wxBitmap& bitmap )
     SetFont(GetFont());
 }
 
-double wxMemoryDCImpl::GetDPIScaleFactor() const
+wxSize wxMemoryDCImpl::GetPPI() const
 {
-    return m_contentScaleFactor;
+    if ( GetWindow() )
+        return GetWindow()->GetDPI();
+    else
+        return wxDisplay::GetStdPPI() * GetContentScaleFactor();
```

Comment:
Sorry, I don't understand this:

1. How can a `wxMemoryDC` have an associated window?
2. Why do we need multiply only the display DPI but not the window one by the scale factor?

Reply:
I added this for a workaround I mentioned before (using `memdc.GetImpl()->SetWindow( )`). And that window already has the correct DPI so no scaling would be needed.

It is probably better to remove it, so I did.